test/libsolidity/syntaxTests/memberLookup/contract_missing_payable_fallback.sol

```diff
@@ -0,0 +1,8 @@
+contract C {
+    function f() public {
+        address(this).send(10);
```

Comment:
Please also add an exmple of transfer.

Acutally I cannot imagine that we do not have tests for that already...